client/components/preview/Preview.js

```diff
@@ -1,10 +1,9 @@
+/* eslint-disable no-shadow */
```

Comment:
I'm not sure what variables are shadowing which, but it definitely can be confusing looking at code where this happens.